mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/impl/ClientState.scala

```diff
@@ -221,9 +223,12 @@ import scala.util.{Failure, Success}
     )
   }
 
+  private val ReceiveConnAck = "receive-connack"
```

Comment:
I'd like to recommend case objects as timer identifiers.

Reply:
Why so? More space used...

Reply:
Cheaper equality, never mind.